addons/calendar/models/calendar.py

```diff
@@ -159,7 +159,7 @@ def copy(self, default=None):
         raise UserError(_('You cannot duplicate a calendar attendee.'))
 
     @api.multi
-    def _send_mail_to_attendees(self, template_xmlid, force_send=False):
+    def _send_mail_to_attendees(self, template_xmlid, force_send=False, recurring_event_id=None):
```

Comment:
Update the docstring. Moreover, the event received here might not be a recurring event, it could be any type of event. Therefore, I think the wording `recurring_event_id` is misleading. Maybe `force_event_id`